config/formats/typescript-export-definition.ts

```diff
@@ -36,22 +43,24 @@ const getTokenType = (tokenTypesPath: string): string => {
  * @param type
  * @returns
  */
-const convertPropToType = (value: string, type: string): string => {
+const convertPropToType = (value: string, type?: string): string => {
   switch (type) {
     case 'color':
       if (value[0] === '#') {
         return 'ColorHex'
       }
       return 'string'
     case 'dimension':
-      if (value.substring(value.length - 3) === 'rem') {
-        return 'SizeRem'
-      }
+      if (typeof value === 'number') return 'number'
```

Comment:
value seems to be typed as a string.. should this be !isNan instead? to account for values like "2"? or should the type defs be updates to accept string and number.

Reply:
I think updating the type definition would be correct here. But technically a `[dimension` must be a string with `px` or `rem`](https://design-tokens.github.io/community-group/format/#dimension) so maybe we can just scrap this line. What do you think?

Reply:
yeah scrap it if it never gets evaluated 👍